code/game/objects/items/RCD.dm

```diff
@@ -492,12 +492,12 @@ ARCD
 	var/floordelay = 10
 	var/decondelay = 15
 
-	var/color_choice = null
+	var/color_choice = "#FFF"
```

Comment:
is this wrong or am i dumb?

Reply:
You're right, null would be black, not white. Probably not necessary to even define to be honest.

Reply:
You were right. I didn't pay attention, thought if it was left null it'd grab black as default but if its null it skips color.